content/_changelogs/9.2.1.md

```diff
@@ -0,0 +1,22 @@
+# Release 9.2.1
+
+_Released 1/10/2022_
+
+**Bugfixes:**
+
+- Fixed a regression in [9.2.0](/guides/references/changelog#9-2-0) to keep the
+  Cypress Runner open after each spec finishes when the `--headed --no-exit`
+  flags are passed to `cypress run`. Fixes
+  [#19485](https://github.com/cypress-io/cypress/issues/19485).
+- Improve heuristic for verifying when the `--openssl-legacy-provider` option
+  should be passed to the plugins' child process when the system Node version is
+  v17+. Fixes [#19320](https://github.com/cypress-io/cypress/issues/19320).
+- Correctly cast the `CYPRESS_VERIFY_TIMEOUT` environment variable as a number.
+  Fixes [#19476](https://github.com/cypress-io/cypress/issues/19476).
```

Comment:
Is there any way to expand on these 2 points to explain how this change will affect the user if they may have been affected. "so that Cypress will no longer...." for example.